kibble/models/film.go

```diff
@@ -137,7 +134,9 @@ func (films *FilmCollection) MakeTitleSlugsUnique() {
 			}
 
 			// find the correct value to update, without making a copy
-			(*films)[group[j]].TitleSlug = fmt.Sprintf("%s-%d", (*films)[group[j]].TitleSlug, j+1)
+			if val, ok := (*films)["/film/" + strconv.Itoa(group[j])]; ok {
```

Comment:
Doesn't seem possible for this `if` to fail - overly cautious or linter? If it is possible to fail surely a warning is called for?

Reply:
I tried both ways and concluded to leave as is, here is my reasoning:

1) We use val twice, so we have to declare val anyway, so may as well take advantage of the safety
```
val := (*films)["/film/"+strconv.Itoa(group[j])]
val.TitleSlug = fmt.Sprintf("%s-%d", val.TitleSlug, j+1)
```

2) I tried tacking on else, but it felt silly as theres such a high chance it can't fail 
```
} else {
  fmt.Println("key not found")
}
```
I hope you see my dilemma